mne/epochs.py

```diff
@@ -1443,11 +1461,12 @@ def _get_data(self, out=True, picks=None, item=None, verbose=None):
                           select_data=False)
 
         if out:
-            if picks is None:
+            if orig_picks is None:
+                data *= ch_factors[np.newaxis, :, np.newaxis]
                 return data
             else:
-                picks = _picks_to_idx(self.info, picks)
-                return data[:, picks]
+                data = data[:, picks] * ch_factors[np.newaxis, :, np.newaxis]
```

Comment:
```suggestion
                data = data[:, picks]
                if units is not None:
                    data *= ch_factors[np.newaxis]
```

Reply:
@larsoner @sappelhoff, should it be done also L1380 and L1465? And also in *base.py* at L850, L852, L862 and L904?

Reply:
Agreed above it could be done. I also updated my comment because I realized I had the broadcasting logic wrong (whoops!), needs to be `[:, np.newaxis]` not `[np.newaxis]` (to replace the PR's `[np.newaxis, :, np.newaxis]). Then this will also match what's in `base.py` (and those shouldn't be changed).